### PR TITLE
acrn-config: Bug fix rel 15

### PR DESCRIPTION
**Pull request by liuwei987**
- acrn-config: enable log_setting in all vm

- check pass-thruogh device for audio/audio_codec

- correct vuart1 setting in scenario config

- add 'tap_' perfix for virtio-net

Tracked-On: #4249
Signed-off-by: Wei Liu weix.w.liu@intel.com
Acked-by: Victor Sun victor.sun@intel.com

### CODE REVIEW

**Comment by wenlingz** [comment]
@binbinwu1 please help approve this PR, which is for release-1.5, thanks

File: misc/acrn-config/launch_config/com.py [hunk 1]
```diff
@@ -541,6 +538,9 @@ def dm_arg_set(names, sel, virt_io, dm, vmid, config):
                 return
         print("   {} \\".format(launch_cfg_lib.PM_CHANNEL_DIC[pm_key]), file=config)
 
+    # set logger_setting for all vm
```
Comment (binbinwu1) [inline misc/acrn-config/launch_config/com.py]: all vm -> all VMs, please correct it both here and the commit message

**Comment by HuiHuangShi** [comment]
No new violations to the coding guideline detected.<br>No New Name Conflict<br><br>No New Function Declaration/Definition Mismatch<br>

**Comment by wenlingz** [comment]
OK to verify

**Comment by HuiHuangShi** [comment]
No new violations to the coding guideline detected.<br>No New Name Conflict<br><br>No New Function Declaration/Definition Mismatch<br>

**Pull request merged**